packages/field-base/test/keyboard-mixin.test.js

```diff
@@ -0,0 +1,43 @@
+import sinon from 'sinon';
+import { expect } from '@esm-bundle/chai';
+import { sendKeys } from '@web/test-runner-commands';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { KeyboardMixin } from '../src/keyboard-mixin.js';
+
+customElements.define(
+  'keyboard-mixin-element',
+  class extends KeyboardMixin(PolymerElement) {
+    static get template() {
+      return html`<div></div>`;
+    }
+  }
+);
+
+describe('keyboard-mixin', () => {
+  let element;
+
+  beforeEach(() => {
+    // Sets tabindex to 0 in order to make the element focusable for the time of testing.
+    element = fixtureSync(`<keyboard-mixin-element tabindex="0"></keyboard-mixin-element>`);
+    element._onKeyDown = sinon.spy();
+    element._onKeyUp = sinon.spy();
```

Comment:
Not a blocker: you could use another approach to align with [this example](https://github.com/vaadin/web-components/blob/86c025abd605d5a4a3c0ae36eb07c34704cee1f2/packages/field-base/test/input-mixin.test.js#L85-L105).

Reply:
Done.